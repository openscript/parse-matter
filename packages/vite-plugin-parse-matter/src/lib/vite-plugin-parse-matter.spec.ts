// import { vitePluginParseMatter } from './vite-plugin-parse-matter';
//
// describe('vitePluginParseMatter', () => {
//   it('should work', () => {
//     expect(vitePluginParseMatter()).toEqual('vite-plugin-parse-matter');
//   });
// });
