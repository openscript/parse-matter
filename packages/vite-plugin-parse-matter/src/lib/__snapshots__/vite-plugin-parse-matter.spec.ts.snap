// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`vitePluginParseMatter > should work 1`] = `
{
  "name": "parse-matter",
  "transform": [Function],
}
`;
